### PR TITLE
[CIAM-535] [KEYCLOAK-18574] [CIAM-690] [CIAM-696] [CIAM-711] [CIAM-743] Initial RH-SSO 7.5.0.GA image

### PR DESCRIPTION
**Pull request by iankko**
[CIAM-535] [KEYCLOAK-18574] [CIAM-690] [CIAM-696] [CIAM-711] [CIAM-743] Initial RH-SSO 7.5.0.GA image

See the particular commits (they have been intentionally kept separated) for additional details about the underlying changes

Corresponding pipeline runs:

Image build / testing: https://keycloak.jenkins.server.com/view/RH-SSO/job/rh-sso-pipeline-continuous_container/205/

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by iankko** [comment]
@drichtarik jFYI:

- There are hardcoded MD5 sums of `maven-repo.zip` & `keycloak-server-overlay.zip` artifacts, 
- And the `keycloak.openshift.clients` module expects the Zip artifacts with the adapters to be prefixed with the `keycloak-` prefix (rather than by the `rh-sso` one, expected by CPaaS)

so the image is buildable using the old pipeline too. But those two points are just easily fixable (for the case of CPaaS build) nuances.

The other changes would apply for CPaaS pipeline too.

File: modules/eap/setup/eap/modules/module.yaml [hunk 1]
```diff
@@ -17,11 +17,11 @@ packages:
...
     # RH-SSO add-on:
     # -------------
-    # Add 'which' & 'tree' RPM packages
-    # "which" tool is handy for debugging issues / troubleshooting
+    # Add 'which', 'tree' & 'vim-minimal' RPM packages, which are handy for
+    # issue debugging / troubleshooting purposes
     - which
-    # "tree" tool is handy to inspect directory structure when needed
     - tree
+    - vim-minimal
```
Comment (pskopek) [inline modules/eap/setup/eap/modules/module.yaml]: Is vim really necessary here? We should keep number of pkgs as low as possible.

Comment (iankko) [reply modules/eap/setup/eap/modules/module.yaml]: Not really. It's handy, but can be installed each time, when the container is run (with `-u root` args). Dropped it from latest version

**Review by pskopek — APPROVED** [review]
used in successful build of rh-sso 7.5.0

**Pull request merged**